debian.sh

```diff
@@ -302,7 +302,7 @@ fi
 #**************************************************************************************************
 # Python Modules
 #**************************************************************************************************
-until sudo pip3 install dnspython monotonic pillow psutil requests tornado wsaccel brotli fonttools selenium future usbmuxwrapper
+until sudo pip3 install dnspython monotonic pillow psutil requests tornado wsaccel brotli fonttools selenium future usbmuxwrapper tzlocal
```

Comment:
https://github.com/catchpoint/WebPageTest.agent/pull/638/files#diff-292d079fe725eed06ead69e3675d638d8bda0c4b65aae36ebde665c7ed52a2beR20

The new logging has also pytz as dependency
```suggestion
until sudo pip3 install dnspython monotonic pillow psutil requests tornado wsaccel brotli fonttools selenium future usbmuxwrapper pytz tzlocal
```